Jax/autoload.php

```
@@ -17,7 +15,7 @@
     $classPath = dirname(__DIR__) . '/' . str_replace('\\', '/', $className) . '.php';
 
     if (!file_exists($classPath)) {
-        throw new Exception("Error loading class: {$classPath}");
+        return;
```

will this silently fail when it can't find a class?